src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -145,7 +145,15 @@ def all_paths(self) -> set[Path]:
 
     def all_relative_names(self) -> set[str]:
         """This method is intended to simplify testing"""
-        return {d.path.relative_to(self._path_lookup.cwd).as_posix() for d in self.all_dependencies}
+        all_names = set[str]()
+        dependencies = self.all_dependencies
+        for path in self._path_lookup.paths:
```

Comment:
@ericvergnaud : Serge changed this to all paths instead of only the `cwd`. What do you think it should be? It breaks a unit test as it returns a relative name `"."` for the file which is a dependency and appended to the sys paths.